ground/src/main/java/com/google/android/ground/repository/LocationOfInterestRepository.kt

```diff
@@ -87,15 +87,22 @@ constructor(
         Single.error { NotFoundException("Location of interest not found $locationOfInterest") }
       )
 
-  fun createLocationOfInterest(geometry: Geometry, job: Job, surveyId: String): LocationOfInterest {
+  fun createLocationOfInterest(
+    geometry: Geometry,
+    job: Job,
+    surveyId: String,
+    ownerEmail: String
+  ): LocationOfInterest {
     val auditInfo = AuditInfo(authManager.currentUser)
     return LocationOfInterest(
       id = uuidGenerator.generateUuid(),
       surveyId = surveyId,
       geometry = geometry,
       job = job,
       created = auditInfo,
-      lastModified = auditInfo
+      lastModified = auditInfo,
+      ownerEmail = ownerEmail,
```

Comment:
We already have current user in this method. Should we use that instead of passing it to this function as a param?

Reply:
Done

Reply:
Done